addons/mail/static/src/components/autocomplete_input/autocomplete_input.js

```diff
@@ -94,6 +94,7 @@ export class AutocompleteInput extends Component {
      */
     _onAutocompleteSelect(ev, ui) {
         if (this.props.select) {
+            ev.preventDefault();
             this.props.select(ev, ui);
         }
     }
```

Comment:
Preventing the default behavior directly from within the component will prevent us from using it in all potential future flows that use this component. You'd better put the `preventDefault` in the first function specific to the flow to avoid that.

Reply:
One last nitpicking and then we should be good, I promise 😄
I feel like `onAddItemAutocompleteSelect` is intended to dispatch the event to the relevant handler based on the "autocompleteMethod", no more, no less, so I wouldn't put the `preventDefault` in there either. I think `handleAddChannelAutocompleteSelect` would be a better place.

Reply:
@oomsveta Done!